lib/SILOptimizer/Transforms/SILLowerAggregateInstrs.cpp

```diff
@@ -9,9 +9,12 @@
 // See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
 //
 //===----------------------------------------------------------------------===//
-//
-// Simplify aggregate instructions into scalar instructions.
-//
+///
+/// \file
```

Comment:
I really don't like the \file tag. Isn't that automatically recognized just like doc comments on declarations?